pyV2DL3/eventdisplay/fillEVENTS.py

```diff
@@ -192,6 +201,6 @@ def __fillEVENTS__(edFileIO, select=None):
             "TSTART": tstart_from_reference,
             "TSTOP": tstop_from_reference,
         },
-        {"azimuth": avAz, "zenith": (90.0 - avAlt), "pedvar": avPedvar},
+        {"azimuth": avAz, "zenith": (90.0 - avAlt), "pedvar": avPedvar, "woffset": Woffset},
```

Comment:
Suggest to explicitly print the wobble directions as (North,East) wobble.

Reply:
Do you mean to print wobble directions as (North, West) wobble, right?